modules/speicher_e3dc/main.sh

```diff
@@ -10,7 +10,7 @@ else
 	MYLOGFILE="${RAMDISKDIR}/bat.log"
 fi
 
-bash "$OPENWBBASEDIR/packages/legacy_run.sh"  "modules.e3dc.device" "bat" "$e3dcip" "$e3dc2ip" "$e3dcextprod" "$pvwattmodul" "1">>"${MYLOGFILE}" 2>&1
+bash "$OPENWBBASEDIR/packages/legacy_run.sh"  "modules.devices.e3dc.device" "bat" "$e3dcip" "$e3dc2ip" "$e3dcextprod" "$pvwattmodul" "1">>"${MYLOGFILE}" 2>&1
```

Comment:
Den Teil habe ich in #2513 als Hotfix korrigiert, damit das aktuelle Modul wieder läuft, bis dieser PR gemerged ist. Kann also nach einem Rebase entfallen.

Reply:
gemacht, geht im #2518 weiter